python/mleap/version.py

```diff
@@ -16,4 +16,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.8.1"
+__version__ = "0.9.4"
```

Comment:
@hollinwilkins maybe we could release an updated pip package once this gets merged.